pipeline-scripts/check-build-promotion.sh

```diff
@@ -0,0 +1,8 @@
+LAST_BUILD=$(
+  drone build ls --format '{{.Number}}' --branch "$DRONE_BRANCH" --status success \
+  --limit 100 "$DRONE_REPO" | head -n 1
+)
+if [ $LAST_BUILD != $DRONE_BUILD_PARENT ]; then
+  echo 'Could not promote as last build: ' $LAST_BUILD ' was not successful';
```

Comment:
Should we reword this? `$LAST_BUILD` will return the last successful build, so the message is misleading